### PR TITLE
Check for duplicate mod ids

### PR DESCRIPTION
**Pull request by AlexejheroYTB**

### CODE REVIEW

File: QModManager/Patcher.cs [hunk 1]
```diff
@@ -346,6 +348,35 @@ private static bool LoadMod(QMod mod)
...
+        private static void RemoveDuplicateModIDs()
+        {
+            List<QMod> duplicateModIDs = new List<QMod>();
+
+            foreach (QMod mod in sortedMods)
+            {
+                List<QMod> matchingMods = sortedMods.Where((QMod qmod) => qmod.Id == mod.Id).ToList();
+                if (matchingMods.Count > 1)
+                {
+                    matchingMods.Do((QMod qmod) =>
```
Comment (PrimeSonic) [inline QModManager/Patcher.cs]: Would it be possible to write this without LINQ?  
I'm not even seeing where anything is being added to `duplicateModIDs`.

Comment (AlexejheroYTB) [reply QModManager/Patcher.cs]: Line 362 is the line you're looking for.  
Also `Do` is not from LINQ, it's from Harmony. (literally)
  
If you want, removing `Do` and using normal C# can be easily achieved.

Comment (PrimeSonic) [reply QModManager/Patcher.cs]: Yeah. For something this critical, I don't mind writing extra lines of code as long as the end result is clear and readable.  
Feel free to use `foreach`. I just want to be able to see exactly when a duplicate is detected and how.

Comment (AlexejheroYTB) [reply QModManager/Patcher.cs]: Addressed in commit 58f17ff

**Pull request merged**